### DISCUSSION TITLE
Updating examples of r2frida commands

### DISCUSSION THREAD

**Discussion by ZabGo** [OWASP/mastg#2446] [Ideas]
In the examples showing how to run r2frida commands, The MASTG is using the old way to start r2frida commands, `\` instead of `:`. 

We should update the guide so that the examples are showing how to run the commands the new way by starting them with `:`. 
old way: `\iE `
new way: `:iE`

**Comment by cpholguera** [comment]
That'd be awesome @ZabGo! I've created a ticket for that: https://github.com/OWASP/owasp-mastg/issues/2448
Would you like me to assign it to you?

**Reply by ZabGo** [reply]
It seems that I was not fast enough to reply haha. I'm glad it's fixed now :-)

**Reply by cpholguera** [reply]
Thank you @ZabGo!
